test/unit/handlers/transfers/handler.test.js

```diff
@@ -1035,7 +1035,7 @@ Test('Transfer handler', transferHandlerTest => {
       TransferService.getById.returns(P.resolve({ condition: 'condition', payeeFsp: 'dfsp2', transferState: TransferState.COMMITTED }))
       TransferService.validateDuplicateHash.returns(P.resolve({}))
       Validator.validateFulfilCondition.returns(true)
-      Util.createPrepareErrorStatus.returns(fulfilMessages[0].value.content.payload)
+      // Util.createPrepareErrorStatus.returns(fulfilMessages[0].value.content.payload)
```

Comment:
Please apply the same comment for ALL similiarly commented out code in this file --> `// Util.createPrepareErrorStatus.returns(fulfilMessages[0].value.content.payload)`

Reply:
All of these occurrences have been removed